Example/Backpack/UIKit/Components/Navigation bar/NavigationBarViewController.swift

```diff
@@ -25,12 +25,22 @@ class NavigationBarViewController: UIViewController {
     var showRightButton: Bool = true
     var collapsed: Bool = false
     var interactive: Bool = false
+    var isOnImage: Bool = false
     
     private static let CellIdentifier = "CellIdentifier"
 
     @IBOutlet weak var navigationButton: UIButton!
     @IBOutlet weak var navigationBar: BPKNavigationBar!
     @IBOutlet weak var tableView: UITableView!
+    
+    private var imageView: UIImageView = {
+        let imageView = UIImageView(frame: .zero)
```

Comment:
Nit: You can create the imageView with an image
`UIImageView(image: UIImage(named: "navigation_bar_image"))`

Saves you a few lines of code :D

Reply:
Updated at [208ccfb](https://github.com/Skyscanner/backpack-ios/pull/1678/commits/208ccfb5829c61dff934c7685f789b945faa7682)